examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -160,6 +224,35 @@ void RvcDevice::HandleOpStateGoHomeCallback(Clusters::OperationalState::GenericO
     }
 }
 
+bool RvcDevice::HandleIsSetSelectedLocationCallback(std::string & locationText)
+{
+    bool canSet;
+
+    switch(mRunModeInstance.GetCurrentMode())
+    {
+    case RvcRunMode::ModeIdle:
+        canSet = true;
+        break;
+
+    case RvcRunMode::ModeCleaning:
+        canSet = false;
+        locationText = "SelectLocations command - cannot select locations while cleaning";
```

Comment:
```suggestion
        locationText = "Cannot select locations while cleaning";
```

I think we don't need the "SelectLocations command" part of the text, here and below, since this will be provided in the SelectLocationsResponse command.